AlbumsNetCoreNoFramework/AlbumsNetCoreNoFramework.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
Can you set the same properties in this csproj as the AlbumsNetCore app?

Reply:
Done